.github/workflows/_rust_lints.yml

```diff
@@ -45,6 +47,18 @@ jobs:
             exit 1    # Fail the workflow
           fi
 
+  examples:
+    if: (!cancelled() && inputs.isRustExample)
+    runs-on: [self-hosted]
+    steps:
+      - uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+      - name: Check docs example
+        run: |
+          for manifest in ./**/examples/**/Cargo.toml; do
+              echo "Checking $manifest"
+              cargo check --manifest-path $manifest --all-features
```

Comment:
We also need fmt, clippy, etc...

Reply:
Alrighty

Reply:
I pushed one possible solution to this. Another I tried is to re-use this workflow for both but it is hard to make reusable workflows work with a loop like that so I don't know if it's better